tests/test_datacube_alchemist.py

```diff
@@ -0,0 +1,40 @@
+from unittest import mock
+import importlib
+
+from datacube_alchemist.worker import deterministic_uuid, get_transform_info, _import_transform, \
+    tile_id_to_date
+
+
+def test_deterministic_uuid():
+    mocked_task = mock.MagicMock()
+    transform = 'wofs.virtualproduct.WOfSClassifier'
+    mocked_task.settings.specification.transform = transform
+    mocked_task.dataset.id = '81ad70a7-c02b-5f48-b10e-4b5d3ab49985'
+    algorithm_version = '1.4'
+    other_tags = {}
+    mocked_task.settings.output.metadata = {}
+    mocked_task.settings.output.metadata['dataset_version'] = '2.2.1'
+    result, uuid_values = deterministic_uuid(mocked_task, algorithm_version='1.4', **other_tags)
+    # print (result)
```

Comment:
Please check the actual return value here. If it does indeed generate a deterministic UUID, we definitely want it in the test in case something causes it to change. :)

Reply:
Yeah, I'll do it.